x/metadata/client/cli/tx.go

```diff
@@ -489,6 +492,46 @@ icon-url           - address to a image to be used as an icon (optional, can onl
 	return cmd
 }
 
+// AddContractSpecToScopeSpecCmd creates an add contract spec to scope spec command
+func AddContractSpecToScopeSpecCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "add-contract-spec-to-scope-spec contract-specification-id scope-specification-id",
+		Short:   "Add an existing contract specification to a scope specification",
+		Aliases: []string{"acstss"},
+		Args:    cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			contractSpecID, err := types.MetadataAddressFromBech32(args[0])
+			if err != nil {
+				return fmt.Errorf("invalid contract specification id : %v", err)
+			}
+			scopeSpecID, err := types.MetadataAddressFromBech32(args[1])
+			if err != nil {
```

Comment:
```suggestion
			if err != nil || !scopeSpecID.IsScopeSpecificationAddress() {
```
verify address type is scope specification. note: will need to handle/create an error for this to use with following line